CommandDotNet/CommandInvoker/CommandInvocation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using CommandDotNet.Models;
+
+namespace CommandDotNet.CommandInvoker
+{
+    public class CommandInvocation

```

Comment:
I think we need to rename ParameterValues & MergedParameters because it's not obvious what they're for.  My naming skills seem to be offline atm, but I was thinking something like:

- ParameterValues -> ArgsFromCli
- MergedParameters -> ParamsForCommandMethod

Reply:
That's a good suggestion